pkg/apis/eventing/v1alpha1/broker_types_test.go

```diff
@@ -60,40 +60,48 @@ func TestBrokerGetCondition(t *testing.T) {
 	}{{
 		name: "single condition",
 		bs: &BrokerStatus{
-			Conditions: []duckv1alpha1.Condition{
-				brokerConditionReady,
+			Status: duckv1alpha1.Status{
```

Comment:
The duplicative nature of these test changes seems like evidence in favor of fluent builders

Reply:
No need to make the test easy, you should never need to make these objects by hand.

Reply:
Um... this is a test in which you had to make these objects by hand! :laughing: